docs/components/PropTable.js

```diff
@@ -59,16 +59,21 @@ function Td({
   shrink?: boolean,
   color?: 'default' | 'subtle',
 |}) {
+  let tdStyle = {
```

Comment:
unrelated to SideNav

Reply:
@AlbertCarreras instead, could we avoid mutating an object (`tdStyle`) and do this logic inline? See fix at https://github.com/pinterest/gestalt/pull/2220/files#diff-0520d70de71d8ac5e7747ab488703c88263481a4b3ce3579d3597cef8a940feaL62

The previous hydration issue was caused by the fact that we didn't always return `null`

```
orderBottom: border && '1px solid #ddd',
```
should have been
```
orderBottom: border ? '1px solid #ddd' : null,
```

and
```
width: shrink ? '1px' : '',
```
should have been
```
width: shrink ? '1px' : null,
```